DYDX-Bots/Roles_Monitor/src/findings.ts

```diff
@@ -0,0 +1,53 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const createFinding = (name: string, args: any, logAddress: string) => {
+  switch (name) {
+    case "RoleAdminChanged":
+      return Finding.fromObject({
+        name: "Admin role has changed",
+        description: `${name} event was emitted`,
+        alertId: "DYDX-18-1",
+        severity: FindingSeverity.Info,
+        type: FindingType.Info,
+        protocol: "dYdX",
+        metadata: {
+          role: args.role.toString(),
+          previousAdminRole: args.previousAdminRole.toString(),
```

Comment:
The following is just an improvement suggestion. The current version is fine. 

Wouldn't using the role name ( ex. `STAKE_OPERATOR_ROLE`) make the finding more readable? 
We can keep in a map every ROLE name with it's corresponding hash, and use it to convert the role arg into a readable string. Wdyt?